### PR TITLE
New deploy

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Review by fernandohds564 — APPROVED** [review]

File: group_vars/all [hunk 1]
```diff
@@ -69,9 +74,9 @@ pkg_version_pydm_opi: "v1.5.7"
...
 pkg_version_trackcpp: "v4.1.0"
 pkg_version_lnls: "v1.0.0"
-pkg_version_pyaccel: "v3.1.1"
+pkg_version_pyaccel: "v3.2.0"
 pkg_version_pymodels: "v1.2.0"
-pkg_version_apsuite: "v2.8.0"
+pkg_version_apsuite: "v2.9.0"
 
 pkg_version_linac_ioc_ps: "v1.2.0"
 pkg_version_linac_opi: "v1.1.3"
```
Comment (fernandohds564) [inline group_vars/all:82]: @xresende, why is this variable doubled here (look at line 59)?

Comment (xresende) [reply group_vars/all:77]: problem introduced with merge! fixed.

**Review by fernandohds564 — APPROVED** [review]

**Review by lerwys — CHANGES_REQUESTED** [review]
Sorry for jumping in, but can I suggest a change? 

I would rather have the `sirius_bbb_version` variable name instead of the `pkg_version_sirius_bbb`.
The former is the role variable name that we override inside group_vars. This is more consistent with the way we name our role variable, by prefixing them with the role name `sirius_bbb`, rather than using the generic prefix `pkg_version`.

This is also done in this way for other roles, such as: qt, epics7, cs-studio and phoebus. see 
https://github.com/lnls-sirius/lnls-ansible/blob/master/group_vars/all#L21
https://github.com/lnls-sirius/lnls-ansible/blob/master/group_vars/all#L50
https://github.com/lnls-sirius/lnls-ansible/blob/master/group_vars/all#L41
https://github.com/lnls-sirius/lnls-ansible/blob/master/group_vars/all#L32

**Comment by lerwys** [comment]
As discussed with @xresende , I'm reverting the changes related to the `sirius_bbb_version` variable

**Review by lerwys — APPROVED** [review]

**Pull request merged**